src/r5py/util/memory_footprint.py

```diff
@@ -113,7 +113,7 @@ def _get_max_memory(max_memory):
 
     Returns
     -------
-    float
+    int
```

Comment:
Changed the return to (rounded) integer because the << operator requires integer to work